### DISCUSSION TITLE
Supabase Integration not completing

### DISCUSSION THREAD

**Discussion by angadveeersb** [vercel/community#3192] [Bug Report]
### Page to Investigate

https://vercel.com/new/clone

### Steps to Reproduce

Under Add Integrations, when I try to add Supabase, it asks me to login and I login successfully to Supabase and it just shows me the dashboard and does not show me the popup to complete the integration process. I've tried using Safari and Chrome and both browsers have this problem!

**Comment by amyegan** [comment]
I'm sorry to hear you're running into problems, @angadveeersb. The Supabase integration is not developed or maintained by Vercel, so I recommend reaching out to [Supabase Support](https://supabase.com/support) for help with it 🙇

**Comment by amyegan** [comment]
The integration is working again! 🙌 

https://github.com/vercel/community/discussions/3194#discussioncomment-6422129
